content/intro-to-storybook/react/en/test.md

```diff
@@ -56,26 +56,6 @@ Add the package as a dependency.
 yarn add storybook-chromatic
 ```
 
-Import Chromatic in your `.storybook/config.js` file.
```

Comment:
We don't need to do this step any more (this is unrelated to CSF)

Reply:
That's already addressed in svelte, the vue and angular will be updated shortly. 👍